[DOCS] Clarifies data frame transform frequency is also for retries (#44229)
elastic/elasticsearch: docs/reference/data-frames/apis/put-transform.asciidoc

  and optionally a `pipeline` id. See <<data-frame-transform-dest>>.

`frequency`::
  (time units) Optional. The interval between checks for changes in the source

>   indices when the {dataframe-transform} is running continuously. If there is

Reviewer: From discussions with @sophiec20 about the difference between data frame transforms with one or many checkpoints, I'm wondering whether this qualification is necessary.

```suggestion
  indices. If there is
```

Reviewer 2: 1. `when .... is running continuously` -- yes, this is valid here.
2. `Also determines the retry interval in the event of transient failures during {dataframe-transform} search or indexing.` -- we recently added a retry to indexing, best to mention both

    (Optional, string) The unique identifier for a <<pipeline,pipeline>>.

`frequency`::
  (Optional, time units) The interval between checks for changes in the source
  indices when the {dataframe-transform} is running continuously. Also determines

>   the retry interval in the event of transient failures during {dataframe-transform}

Reviewer: ```suggestion
  the retry interval in the event of transient failures while the {dataframe-transform} is
```

>   search or indexing. The minimum value is `1s` and the maximum is `1h`. The

Reviewer: ```suggestion
  searching or indexing. The minimum value is `1s` and the maximum is `1h`. The
```
